packages/x-date-pickers/src/locales/beBY.ts

```diff
@@ -0,0 +1,77 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { DateView } from '../internals/models';
+
+const beBYPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: 'Папярэдні месяц',
+  nextMonth: 'Наступны месяц',
+
+  // View navigation
+  openPreviousView: 'адкрыць папярэдні выгляд',
+  openNextView: 'адкрыць наступны выгляд',
+  calendarViewSwitchingButtonAriaLabel: (view: DateView) =>
+    view === 'year'
+      ? 'гадавы выгляд адкрыты, перайсці да каляндарнага выгляду'
+      : 'каляндарны выгляд адкрыты, перайсці да гадавога выгляду',
+  inputModeToggleButtonAriaLabel: (isKeyboardInputOpen, viewType) =>
+    isKeyboardInputOpen
+      ? `тэкставае поле адкрыта, перайсці да  ${viewType} выгляду`
```

Comment:
Could you refer to `deDE` or any other locale having the additional view translation handling to solve this leftover english value?
![Screenshot 2023-01-06 at 11 29 11](https://user-images.githubusercontent.com/4941090/210972812-653f3599-a20a-446a-a2f4-64086d614e34.png)

Reply:
Thank you!